firebase-app.html

```diff
@@ -69,22 +69,34 @@
           },
 
           /**
+           * The URL of your Firebase Storage Bucket. You can find this
+           * URL in the Database panel of the Firebase Console.
+           * Available on the Firebase Console.
+           *
+           * For example: 'polymerfire-test.appspot.com/'
+           */
+          storageUrl: {
```

Comment:
This should have a default empty value, shouldn't it? the other properties are required for a functioning firebase app, this one is not. If we don't default it, I guess the `app` property is never computed unless we set `storageUrl`.